content/en/docs/zero-code/java/_index.md

```diff
@@ -10,6 +10,6 @@ cascade:
     otel: 1.47.0
 ---
 
-Zero-code instrumentation with Java uses a Java agent JAR or Spring Boot
-Starter. To learn how to manually instrument your service or app code, see
+Zero-code instrumentation with Java uses a Java agent JAR, Spring Boot Starter,
+or Quarkus. To learn how to manually instrument your service or app code, see
```

Comment:
```suggestion
Common options for zero-code instrumentation with Java are the Java agent JAR, Spring Boot Starter,
or the Quarkus OpenTelemetry Extension. To learn how to manually instrument your service or app code, see
```

Can we write this more open ended (since there are options outside these 3, that we are not (yet) listing here)

Reply:
I think "common options" imply that there might be others. Sounds good enough for me, for now.